tests/nativeapp/test_children.py

```diff
@@ -140,13 +139,13 @@ def test_children_bundle_with_custom_dir(project_directory):
             setup_script_content = f.read()
             custom_dir_path = Path("_entities", "my_streamlit")
             assert setup_script_content.endswith(
-                dedent(
-                    f"""
-                    -- AUTO GENERATED CHILDREN SECTION
-                    CREATE OR REPLACE STREAMLIT v_schema.my_streamlit FROM '{custom_dir_path}' MAIN_FILE='streamlit_app.py';
-                    CREATE APPLICATION ROLE IF NOT EXISTS my_app_role;
-                    GRANT USAGE ON SCHEMA v_schema TO APPLICATION ROLE my_app_role;
-                    GRANT USAGE ON STREAMLIT v_schema.my_streamlit TO APPLICATION ROLE my_app_role;
-                    """
-                )
+                f"""
+-- AUTO GENERATED CHILDREN SECTION
+CREATE OR REPLACE STREAMLIT IDENTIFIER('v_schema.my_streamlit')
+FROM = '{custom_dir_path}'
+MAIN_FILE = 'streamlit_app.py';
+CREATE APPLICATION ROLE IF NOT EXISTS my_app_role;
+GRANT USAGE ON SCHEMA v_schema TO APPLICATION ROLE my_app_role;
+GRANT USAGE ON STREAMLIT v_schema.my_streamlit TO APPLICATION ROLE my_app_role;
+"""
```

Comment:
Why no `dedent`?

Reply:
I left it out while fixing this test, but it seems it was not the problem. Fixed.